packages/flutter/lib/src/services/debug.dart

```diff
@@ -13,6 +13,18 @@ import 'hardware_keyboard.dart';
 /// of their extent of support for keyboard API.
 KeyDataTransitMode? debugKeyEventSimulatorTransitModeOverride;
 
+/// Profile and print statistics on Platform Channel usage.
+///
+/// When this is is true statistics about the usage of Platform Channels will be
+/// printed out periodically to the console and Timeline events will show the
+/// time between sending and receiving a message (encoding and decoding time
+/// excluded).
+///
+/// The statistics include the total bytes transmitted and the average number of
+/// bytes per invocation in the last quantum.  "Up" means in the direction of
+/// Flutter to the host platform, "down" is the host platform to flutter.
```

Comment:
Nit: extra space at `. "Up"`

It's also upbytes / downBytes now?

Reply:
> Nit: extra space at . "Up"

done

> It's also upbytes / downBytes now?

yea